gerrit-report.py

```diff
@@ -15,7 +15,7 @@ def query(*args):
     s = subprocess.getoutput("ssh openbmc.gerrit gerrit query " +
                              "--format json --all-reviewers " +
                              "--dependencies --current-patch-set -- '" +
-                             " ".join(args) + "'")
+                             " ".join(args) + "'" + " 2>/dev/null")
```

Comment:
Rather than adding this shell invocation, which likely only works on bash/sh variants, we should probably change from using getoutput to using check_output.  I believe by default this would redirect stderr to no where.